### PR TITLE
Issue#56: elb no longer optional

### PR DESCRIPTION
**Pull request by lukaszraczylo**
Added fix and test for missing ELB definition as per issue#56

### CODE REVIEW

**Comment by mattmb** [comment]
LGTM :+1:

Just need to remove that extra commit that's somehow crept in :-)

File: .eggs/README.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+This directory contains eggs that were downloaded by setuptools to build, test, and run plug-ins.
+
+This directory caches those eggs to prevent repeated downloads.
+
+However, it is safe to delete this directory.
+
```
Comment (mattmb) [inline .eggs/README.txt]: Also I'm guessing you didn't mean to include this ;-)

**Comment by lukaszraczylo** [comment]
Removed 4d0a5d8 from branch "issue-56-elb-no-longer-optional"

**Comment by lukaszraczylo** [comment]
Please ignore - opened new pull requested with cherry picked commit as it was impossible to remove mentioned commit. New PR: https://github.com/ministryofjustice/bootstrap-cfn/pull/64

**Pull request closed without merging**